perfzero/lib/setup.py

```diff
@@ -136,6 +136,30 @@ def _create_docker_image(FLAGS, project_dir, workspace_dir,
   setup_execution_time['build_docker'] = time.time() - docker_start_time
 
 
+def _add_local_includes(dir_list_string, dst_base_dir):
```

Comment:
nits: it seems like a util method that can be put in perfzero/utils.py. Would it be simpler to name this method something like `copy_dir`? `_add_local_includes` carries some semantics that only applies to the caller but not this method.

Reply:
Done. Thanks!

Reply:
Done - thanks!